clippy_lints/src/map_clone.rs

```diff
@@ -105,6 +116,8 @@ impl<'tcx> LateLintPass<'tcx> for MapClone {
             }
         }
     }
+
+    extract_msrv_attr!(LateContext);
```

Comment:
Seeing this PR, I have the suspicion, that this is not always added when new MSRVs get introduced. I will have to check that across Clippy. Same thing with the lint names in the conf.rs docs.

Reply:
I would think that the conf.rs docs are quite up-to-date, but the `extract_msrv_attr` might not be. Now the question is if I'm going to look into it and procrastinate a bit more...

Reply:
Found one without the macro!

Reply:
Sounds like an internal lint, ya dig?

Reply:
Could be, we could probably just check if the struct has a `msrv: Option<RustcVersion>,` field to determine if it should use the macro :thinking:

Reply:
And a second one, the other lint passes all have that macro :upside_down_face:

Reply:
Now I've checked, all lints are now listed in the `conf.rs`. Only `manual_bits` and `borrow_as_prt` were missing.